src/libsyntax/parse/parser/item.rs

```diff
@@ -98,7 +98,7 @@ impl<'a> Parser<'a> {
         if self.eat_keyword(kw::Use) {
             // USE ITEM
             let item_ = ItemKind::Use(P(self.parse_use_tree()?));
-            self.expect(&token::Semi)?;
+            self.expect_semi()?;
```

Comment:
I assume you grepped for the various places?

Reply:
Yep.